lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -849,10 +852,13 @@ void SessionState::get_updates(
     UpdateSessionRequest* update_request_out,
     std::vector<std::unique_ptr<ServiceAction>>* actions_out,
     SessionStateUpdateCriteria* session_uc) {
-  if (curr_state_ != SESSION_ACTIVE) return;
-  get_charging_updates(update_request_out, actions_out, session_uc);
-  get_monitor_updates(update_request_out, session_uc);
-  get_event_trigger_updates(update_request_out, session_uc);
+  if (curr_state_ == SESSION_ACTIVE || curr_state_ == ACTIVE) {
```

Comment:
the logic has changed here. Before we were reporting always, except when SESSION_ACTIVE
Now we are not reporting when `SESSION_ACTIVE` and `ACTIVE`.

Is this intended?

Reply:
Previously the logic like this:
```
if (curr_state_ != SESSION_ACTIVE) return;
```
Means Only if state is `SESSION_ACTIVE` then reporting. Now I have added if state is `SESSION_ACTIVE` or `ACTIVE` (for 5g) then reporting.